src/Sulu/Bundle/WebsiteBundle/Resources/config/services.xml

```diff
@@ -28,6 +28,15 @@
     </parameters>
 
     <services>
+        <service id="sulu_website.default_controller" class="Sulu\Bundle\WebsiteBundle\Controller\DefaultController">
+            <tag name="container.service_subscriber" />
+            <tag name="controller.service_arguments" />
```

Comment:
Would add here `<tag name="sulu.context" context="website" />` as this controller is only available in website context.